configure.ac

```diff
@@ -6,7 +6,7 @@
 # Use and distribution licensed under the BSD license.  See
 # the COPYING file in this directory for full text.
 
-AC_INIT([libdrizzle],[5.1.4],[http://bugs.launchpad.net/drizzle],[libdrizzle],[http://drizzle.org/])
```

Comment:
There is always the extremely ugly option of generating `configure.ac` but I guess it would be too meta, not sure it worth the trouble.